razorpay-payments.php

```diff
@@ -361,16 +355,28 @@ function getOrderCreationData($orderId)
         /**
          * Generates the order form
          **/
-        function generateOrderForm($redirectUrl, $json)
+        function generateOrderForm($redirectUrl, $data)
         {
-            $html = <<<EOT
-<script src="https://checkout.razorpay.com/v1/checkout.js"></script>
-<script>
-    var data = $json;
-</script>
+            wp_enqueue_script('razorpay_wc_script', plugin_dir_url(__FILE__)  . 'script.js',
```

Comment:
@captn3m0 Shouldn't we register the script first and then enqueue it? 
http://www.wpbeginner.com/wp-tutorials/how-to-properly-add-javascripts-and-styles-in-wordpress/